Description: Add missing includes
 Fix FTBFS with the upcoming GCC 4.4
Forwarded: http://xplanet.svn.sf.net/viewvc/xplanet?view=rev&revision=157
Bug-Debian: http://bugs.debian.org/505013
Author: Ruben Molina <rmolina@udea.edu.co>
Index: xplanet-1.2.1/src/Satellite.cpp
===================================================================
--- xplanet-1.2.1.orig/src/Satellite.cpp	2010-12-02 10:22:15.857716544 -0500
+++ xplanet-1.2.1/src/Satellite.cpp	2010-12-02 10:22:29.529716897 -0500
@@ -1,4 +1,5 @@
 #include <clocale>
+#include <cstdio>
 #include <cstring>
 #include <iostream>
 using namespace std;
Index: xplanet-1.2.1/src/Separation.cpp
===================================================================
--- xplanet-1.2.1.orig/src/Separation.cpp	2010-12-02 10:22:15.837716446 -0500
+++ xplanet-1.2.1/src/Separation.cpp	2010-12-02 10:22:29.529716897 -0500
@@ -1,4 +1,5 @@
 #include <cmath>
+#include <cstdio>
 #include <iostream>
 
 #include "Separation.h"
